static/app/views/usageStats/utils.tsx

```diff
@@ -102,3 +109,34 @@ export function isDisplayUtc(datetime: DateTimeObject): boolean {
   const hours = parsePeriodToHours(interval);
   return hours >= 24;
 }
+
+/**
+ * Duplicate of getInterval from static/app/components/charts/utils
+ * except that we do not support <1h granularity for time ranges >6h
+ */
+export function getUsageInterval(datetimeObj: DateTimeObject) {
```

Comment:
Is this something we could share with sessions as I think it has similar restrictions on granularity.

Reply:
Good call. Found the exact function in Sessions and I've moved them to `static/app/components/charts/utils.tsx`